vagrant_repo/data_bags/cerner_splunk/passwords.json

```diff
@@ -1,8 +1,8 @@
 {
   "id": "passwords",
   "bindacct": {
-    "encrypted_data": "b02WiljdA3cQi6rhQYbgzMkykhxO8CNPDLpAJHCp73ff40T8kyPGkReAzMZD\nlFee\n",
-    "iv": "LdcFVALEcfeynhPW+u8xmg==\n",
+    "encrypted_data": "MeZHfvpYSAI8twV96Shq0yuFwP39tuD6u29HdNGXk2/uTM+yvYscQHpyVxnt\nxuj1\n",
```

Comment:
Does the heavy forwarder also make use of the chef-vault passwords? 

If so, we should have also modified the _keys databag. But if not, this doesn't need to change.